packages/core-sdk/src/types/resources/ipAsset.ts

```diff
@@ -321,7 +366,12 @@ export type DistributeRoyaltyTokens = {
   txOptions?: Omit<TxOptions, "encodedTxDataOnly">;
 };
 export type RoyaltyShare = {
+  /** The address of the recipient. */
   recipient: Address;
+  /**
+   *  The percentage of the royalty share.
+   *  @example 10 represents 10% which is 100_000_00.
```

Comment:
`@example` should be for just the example value. You can do this. 
```suggestion
   *  The percentage of the total royalty share. For example, a value of 10 represents 10% of max royalty shares, which is 10,000,000
   * 
   *  @example 10
```